rosey/src/runners/builder/html.rs

```diff
@@ -479,11 +563,15 @@ impl<'a> RoseyPage<'a> {
         }
     }
 
-    pub fn rewrite_meta_tags(&mut self, relative_path: &Path) {
+    pub fn rewrite_meta_tags(
+        &mut self,
+        relative_path: &Path,
+        original_relative_path: &Path,
+        url_translations: &BTreeMap<String, RoseyTranslation>,
+    ) {
         let locale_key = self.get_locale_key();
 
-        let path = relative_path.display().to_string();
-        let path = path.trim_end_matches("index.html");
+        let path = filepath_to_output_url(&relative_path.to_slash_lossy());
```

Comment:
Looks like this isn't being used anywhere, in which case you could also get rid of `relative_path` from this function's arguments